### PR TITLE
Feature 54 group management

### PR DESCRIPTION
**Pull request by kaeline**
https://github.com/Metatavu/rapurc-api-spec/issues/54

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -4302,3 +4807,71 @@ components:
...
+        - email
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+        email:
+          type: string
+          description: user email address
+        groupId:
```
Comment (jarinykanen) [inline swagger.yaml:4847]: Should this be marked as required?

File: swagger.yaml [hunk 2]
```diff
@@ -4302,3 +4807,71 @@ components:
...
```
Comment (jarinykanen) [inline swagger.yaml:4879]: Should this be marked as required?

File: swagger.yaml [hunk 3]
```diff
@@ -3677,6 +3677,511 @@ paths:
...
+      responses:
+        "204":
+          description: Success
+        "default":
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
```
Comment (jarinykanen) [inline swagger.yaml:3998]:
```suggestion
```

**Review by jarinykanen — APPROVED** [review]

File: swagger.yaml (hunk 2 above)
Comment (kaeline) [reply swagger.yaml:4879]: Done

File: swagger.yaml (hunk 1 above)
Comment (kaeline) [reply swagger.yaml:4847]: Done

**Pull request merged**